packages/components/src/TitleBlockZen/_docs/TitleBlockZen.stories.tsx

```diff
@@ -97,76 +98,30 @@ export const Playground: Story = {
   },
 }
 
-// export const WithBadge: Story = {
```

Comment:
These were a bunch of commented out stories, I figure we don't need them any more